openretina/data_io/base.py

```diff
@@ -84,7 +88,11 @@ def get_n_neurons_per_session(responses_dict: dict[str, ResponsesTrainTestSplit]
 def normalize_train_test_movies(
     train: Float[np.ndarray, "channels train_time height width"],
     test: Float[np.ndarray, "channels test_time height width"],
-) -> tuple[Float[np.ndarray, "channels train_time height width"], Float[np.ndarray, "channels test_time height width"]]:
+) -> tuple[
+    Float[np.ndarray, "channels train_time height width"],
+    Float[np.ndarray, "channels test_time height width"],
+    dict[str, float | None],
```

Comment:
```suggestion
    dict[str, float],
```

Don't we always return a float here?

Reply:
but I see now why you did it. If it's much work to get the type annotations consistent in lateron code you can keep it like that